CHANGELOG.rst

```diff
@@ -2,10 +2,20 @@
 History
 =======
 
-0.16.0 (unreleased)
+0.16.0 (2023-02-09)
 ===================
 
-*
+* Add support to export and import configuration data from plugins.
+
+* Add a fixture ``alfasim_runner`` (add to pytest relevant configuration ``"alfasim_sdk.testing.fixtures"`` to use it) to allow plugin authors to run their plugin against an installed version of ALFAsim in a test environment.
```

Comment:
What is the pattern, with or without double quotes? (I don't remember)
```suggestion
* Add a fixture ``alfasim_runner`` (add to pytest relevant configuration ``alfasim_sdk.testing.fixtures`` to use it) to allow plugin authors to run their plugin against an installed version of ALFAsim in a test environment.
```

Reply:
I used double quotes because I extracted the from a `conftes.py` and there it is listed as string, but you are right, it could be present in the command line or elsewhere.